codemod/spotlight_util.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2007-2008 Facebook
+
+from Cocoa import *
```

Comment:
Hmmm, my thought here is maybe you're running the python from brew instead of /usr/local/bin?  I'll check on that though.  Is there a way to determine if a module is available before trying to import it?  Can I put an import in a try/catch?